profiles/generators/PreRaids/PR_Generate_Warrior.simc

```diff
@@ -1,79 +1,78 @@
+neck=emergency_stop_keychain,id=221077,ilevel=5938
+shoulders=swarm_monarchs_spaulders,id=221155,ilevel=593
+back=anvilhide_cape,id=221088,bonus_id=ilevel=593
+chest=experimental_goresilk_chestguard,id=221161,ilevel=593,enchant=crystalline_radiance_2
+hands=devourers_gauntlets,id=221151,ilevel=593
+waist=girdle_of_somber_ploys,id=221133,ilevel=593
+legs=legplates_of_duality,id=221173,ilevel=593,enchant=defenders-armor-kit_2
+feet=devoted_plate_walkers,id=221123,ilevel=593
+finger1=85_year_tenure_ring,id=221198,ilevel=593,enchant=radiant_mastery_2
+finger2=wicks_golden_loop,id=221099,ilevel=593,enchant=radiant_mastery_2
+trinket1=skarmorak_shard,id=219300,ilevel=593
+trinket2=ara_kara_sacbrood,id=219314,ilevel=593
+main_hand=poleaxe_of_somber_fate,id=221111,ilevel=593,enchant=authority_of_radiant_power_2
+off_hand=poleaxe_of_somber_fate,id=221111,ilevel=593,enchant=authority_of_radiant_power_2
 
-# save=PR_Warrior_Fury.simc
+save=PR_Warrior_Fury.simc
 
-# warrior="PR_Warrior_Protection"
-# level=70
-# race=draenei
-# spec=protection
-# role=tank
-# position=front
-# talents=3122011
+warrior="PR_Warrior_Protection"
+level=80
+race=goblin
+spec=protection
+role=tank
+position=front
+talents=CkEAAAAAAAAAAAAAAAAAAAAAAYEDAAAAzMzYmZGMbzsMzMz2mZMMNzgZmBwyADbMzMwDMzDMMAAAAAAgZGAgltNADDsBLLGNmBwsFbYD
 
-# head=shadowghast_helm,id=171415,bonus_id=6647/6649/6716/8130/1588/6935,gem_id=173129
-# neck=worldkiller_iris,id=189859,bonus_id=4800/4786/1808/1524,gem_id=173128
-# shoulders=pauldrons_of_the_infinite_infantry,id=188941,bonus_id=1505/7187
-# back=shroud_of_the_sires_chosen,id=189847,bonus_id=7187/1524
-# chest=breastplate_of_the_infinite_infantry,id=188938,bonus_id=1505/7187,enchant=eternal_stats
-# wrists=vambraces_of_the_infinite_infantry,id=188943,bonus_id=1498/7187/6935,gem_id=173128
-# hands=grasps_of_the_infinite_infantry,id=188937,bonus_id=1505/7187,enchant=eternal_strength
-# waist=girdle_of_the_soulflayer,id=179326,bonus_id=1592/6536/6646/6935,gem_id=173128
-# legs=legplates_of_the_infinite_infantry,id=188940,bonus_id=7187/1505
-# feet=shadowghast_sabatons,id=171413,bonus_id=6647/6649/6716/6969/1588
-# finger1=bloodoath_signet,id=178871,bonus_id=1592/6536/6646/6935,gem_id=173128,enchant=tenet_of_critical_strike
-# finger2=ritual_commanders_ring,id=178781,bonus_id=3536/4786/1592/6935,gem_id=173128,enchant=tenet_of_critical_strike
-# trinket1=the_first_sigil,id=188271,bonus_id=1524/7187
-# trinket2=ticking_sack_of_terror,id=185844,bonus_id=1595/6536/6646
-# main_hand=devastated_worldshard,id=189851,bonus_id=7187/1524,enchant=celestial_guidance
-# off_hand=ruined_crest_of_lordaeron,id=189845,bonus_id=7187/1524
+head=waxsteel_greathelm,id=221100,ilevel=593
+neck=strapped_rescue_keg,id=221060,ilevel=593
+shoulders=flesh_connectors_epaulets,id=221185,ilevel=593
+back=anvilhide_cape,id=221088,ilevel=593
+chest=slashproof_business_plate,id=221069,ilevel=593,enchant=crystalline_radiance_2
+wrists=vociferous_subjugators_bracers,id=221167,ilevel=5938
```

Comment:
I'd like to have those Wrists, pleasea

Reply:
😅 ah my bad. Should be fixed now

Reply:
DMed w Archimtiros, fixing typos and a changing a few items.